testdata/p4_14_samples_outputs/06-SimpleVlanStack-midend.p4

```diff
@@ -34,12 +38,43 @@ parser ParserImpl(packet_in packet, out headers hdr, inout metadata meta, inout
         }
     }
     @name(".parse_vlan_tag") state parse_vlan_tag {
-        packet.extract<vlan_tag_t>(hdr.vlan_tag.next);
-        transition select(hdr.vlan_tag.last.ethertype) {
-            16w0x8100 &&& 16w0xefff: parse_vlan_tag;
+        packet.extract<vlan_tag_t>(hdr.vlan_tag[32w0]);
+        transition select(hdr.vlan_tag[32w0].ethertype) {
+            16w0x8100 &&& 16w0xefff: parse_vlan_tag1;
+            default: accept;
+        }
+    }
+    @name(".parse_vlan_tag") state parse_vlan_tag1 {
```

Comment:
@VolodymyrPeschanenkoIntel  It is probably a good idea to preserve the annotation only for the original state but do not add it to the follower states. This might lead to issues with the control plane otherwise